source/uwp/AdaptiveCardTestApp/Views/TestResultView.xaml.cs

```diff
@@ -25,30 +25,47 @@ private void TestResultView_DataContextChanged(FrameworkElement sender, DataCont
             {
                 _currModel = model;
 
-                if (model.Status == TestStatus.Passed)
+                if (model.Status.NewCard)
                 {
-                    ButtonSaveAsExpected.Visibility = Visibility.Collapsed;
+                    // For a new card, only let them update "all"
+                    ButtonUpdateImage.Visibility = Visibility.Collapsed;
+                    ButtonUpdateJson.Visibility = Visibility.Collapsed;
+                    ButtonUpdateOriginals.Visibility = Visibility.Collapsed;
+                    ButtonUpdateAll.Visibility = Visibility.Visible;
```

Comment:
One thing I like to do for UIs like this is first set all the buttons to Collapsed, and then inside each logic branch, all you have to worry about is showing buttons, rather than both showing and hiding. Makes it a little cleaner (and reduces the chance of forgetting to collapse a button). But just a small nit :)

Reply:
Sure

---
In reply to: [292200213](https://github.com/microsoft/AdaptiveCards/pull/3046#discussion_r292200213) [](ancestors = 292200213)